internal/version/version.go

```diff
@@ -0,0 +1,4 @@
+package version
+
+// Version is the current version
+const Version = "1.0.0"
```

Comment:
Should this start at 1.0?

Version also does not seem to be used anywhere in the application.  Is this file needed?

One place it could be used is to log the version to help with debugging.

Reply:
+1, not really seeing the need from this package. Also, it seems like were on version `0.15.0` not v1

Reply:
ok, will remove it from the moment. Also the versioning will be confusing as it will produce an artifact tagged as v3, so I presume the software should start at that version really.

Reply:
done